extensions/wrapper/wrapper/src/main/java/de/sovity/edc/ext/wrapper/api/ui/pages/contract_agreements/services/ContractAgreementDataFetcher.java

```diff
@@ -76,10 +79,44 @@ public List<ContractAgreementData> getContractAgreements(DSLContext dsl) {
             .toList();
     }
 
-    private @NotNull Map<String, SovityContractTerminationRecord> fetchTerminations(DSLContext dsl, List<ContractAgreement> agreements) {
+    @NotNull
+    public ContractAgreementData getContractAgreement(DSLContext dsl, String contractAgreementId) {
 
```

Comment:
nit: kinda weird blank line

Reply:
I can move it after the `public`, but this is the expected location for methods annotations.

Reply:
ok, that one I don't care.
Empty line removed.